scripts/smoke_test.sh

```diff
@@ -8,14 +8,47 @@ SNFORGE_PATH="$2"
 SNCAST_PATH="$3"
 REPO_URL="$4"
 REVISION="$5"
+VERSION="$6"
 
-# Check forge
+# Check forge std from github
 
-$SNFORGE_PATH init my_project
-pushd my_project || exit
+$SNFORGE_PATH new my_project_0
+pushd my_project_0 || exit
 scarb add --dev snforge_std --git "$REPO_URL" --rev "$REVISION"
 $SNFORGE_PATH test || exit
 popd || exit
+scarb cache clean
+
+# Check forge std from registry prebuild
```

Comment:
nit: let's improve on the comments, `Check forge std from registry prebuild` vs `Check forge std from registry build` is not obvious at all